extensions/wikia/UserLogin/UserLoginSpecialController.class.php

```diff
@@ -60,7 +60,7 @@ private function initializeTemplate() {
 		$this->response->addAsset(
 				( $this->wg->request->getInt( 'recover' ) === 1 || empty( $this->wg->EnableFacebookClientExt ) ) ?
 					'userlogin_js_wikiamobile' :
-					'userlogin_js_wikiamobile_fbconnect'
+					'userlogin_facebook_js_wikiamobile'
```

Comment:
The `userlogin_js_wikiamobile` group was being included in `userlogin_js_wikiamobile_fbconnect`, but it's not in `userlogin_js_wikiamobile_fbconnect`. The result is that `/extensions/wikia/UserLogin/js/UserLoginSpecial.wikiamobile.js` isn't being loaded on the login page anymore. 

I think the logic here should be changed to always include `userlogin_js_wikiamobile`, since that's what was happening before anyway, and optionally include `userlogin_facebook_js_wikiamobile`.

Or you could rearrange the assets manager groups to reflect this logic.